lib/pure/streams.nim

```diff
@@ -145,6 +145,10 @@ proc write*[T](s: Stream, x: T) =
   shallowCopy(y, x)
   writeData(s, addr(y), sizeof(y))
 
+proc write*(f: File; x: seq) =
+  ## writes the seq `x` to the the stream `s`.
+  if x.len > 0: writeData(f, unsafeAddr x[0], x.len)
```

Comment:
writeData(f, unsafeAddr x[0], x.len*sizeof(x[0]))

Reply:
Thanks for fast review data-man. Let me do some testing and I will change it. (it seems weird to me bc I copied the code from the next proc)

Reply:
In the other proc the argument is a string, and for a string `.len` is both the number of elements and the number of bytes (since each `char` is a byte).

Reply:
thx for explaining GULPF

Reply:
This PR requires further refinement.
And if x is a seq of strings (or seqs)?
Or x is a seq of tuple with a string fields?

Reply:
I see. Since I don't know how to do that I will close the PR.